src/include/daos/security.h

```diff
@@ -10,7 +10,16 @@
 #include <stdint.h>
 #include <stddef.h>
 #include <daos_types.h>
+#include <daos_prop.h>
 
+/**
+ * Structure representing a resource's ownership by user and group,
+ * respectively.
+ */
+struct ownership {
+	char *user;	/** name of the user owner */
+	char *group;	/** name of the group owner */
+};
```

Comment:
If this structure is to be public then it needs renaming to avoid namespace pollution with applications.